rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/DiscoverJsfSourceFileRuleProvider.java

```diff
@@ -36,9 +36,9 @@ public void perform(GraphRewrite event, EvaluationContext context, FileLocationM
                     }
                 })
                 .where("taglib1")
-                    .matches("(java\\.sun\\.com/jsf/core)|(java\\.sun\\.com/jsf/html)|(http://java\\.sun\\.com/jsf/facelets)")
+                    .matches("(java\\.sun\\.com/jsf/)|(java\\.sun\\.com/jsf/)|(xmlns\\.jcp\\.org/jsf)")
```

Comment:
Isn't `"(java\\.sun\\.com/jsf/)|(java\\.sun\\.com/jsf/)|`duplicate now?

Reply:
I have merged it with the fix

Reply:
Yes, of course. Thanks for catching and fixing that. :)